### ISSUE TITLE
Page titles for application should include name of listing 

### ISSUE THREAD

**Issue by slowbot** [bloom-housing/bloom#2204]
**What is this feature or what feature is this part of?**
Screen readers use page titles to establish context, and the current page title in the public application is just the name of the housing portal.

**Describe the solution you'd like**
Add the listing name to the page title to let applicants know what listing they are applying to. The format should be:
("Name of Listing" Application | Existing Name of Jurisdiction Portal)

**What is the acceptance criteria/definition of done?**
A screen reader should read out name of listing as part of page title

**QA Review Instructions**
This is to be filled out by the developer who completes this issue, before passing to QA.

**Comment by slowbot** [comment]
@emilyjablonski For your review

**Comment by emilyjablonski** [comment]
@seanmalbert We're looking for your feedback here on approach before moving forward if you get a chance

**Comment by seanmalbert** [comment]
@emilyjablonski ,
My original thought was to use `AppSubmissionContext`, which stores the current listing a user is applying for and the forms layout could be updated to check for this and pass title as a prop to the main application layout. This would all work well, except that the screen reader announces the page title right away, so we can't rely on javascript to update the page title. There are ways to hack around this, but there is a much simpler answer, if all we care about is the listing title being announced by the screen reader. In `ui-components/src/blocks/FormCard.tsx`, if we update the `h5` header to be an `h1`, then the screen reader should properly announce the listing title as a user moves through the application.

Do you see any downsides to this?

**Comment by slowbot** [comment]
@seanmalbert I think this is a valid solution for now but would like to get our accessibility consultant to weigh in when they do their review

I the meantime we can close this. I started a new issue which we can validate https://github.com/bloom-housing/bloom/issues/2671
